app/models/post.rb

```diff
@@ -1,3 +1,14 @@
 class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
```

Comment:
**OPTIONAL**
- It'd be nice if you created a custom method using the `after_save` rails callback to call the `udpate_user_posts_counter` method every time the model gets updated!

Ex:
`after_save :udpate_user_posts_counter`

> Please consider this suggestion in the other models

Reply:
Thanks @MahmoudBakr23 , for introducing me to Active Record callbacks! :pray: